src/server/chromium/crNetworkManager.ts

```diff
@@ -288,8 +288,10 @@ export class CRNetworkManager {
     const timingPayload = responsePayload.timing!;
     let timing: network.ResourceTiming;
     if (timingPayload) {
+      const requestSentTime = Math.min(request._timestamp, timingPayload.requestTime);
       timing = {
-        startTime: (timingPayload.requestTime - request._timestamp + request._wallTime) * 1000,
```

Comment:
`startTime` will now be `request._wallTime * 1000)`, but we need to recompute all the timings relative to it.